### ISSUE TITLE
Alert spam for message "Clock Skew Alert Resolved" 

### ISSUE THREAD

**Issue by CharlotteRose** [yugabyte/yugabyte-db#8426]
**Issue**
Customer is seeing an unprecedented number of emails, alerting on `clock-skew` - meanwhile the `Health-Check` is reporting "OK"

**Steps to Reproduce**
This issue is being opened as we have not been able to reproduce thus far. 

**Additional Information**
Below is a copy of the information contained in the alert emails
```
From:	YW-Dev@<domain>.com
Sent:	Tuesday, April 13, 2021 9:02 PM
To:	YugabyteOperations
Subject:	Yugabyte Platform Alert - <[yw-dev][dev]>

Clock Skew Alert for <Universe_Name> is firing.
```

These alert emails are shortly followed by emails stating this issue has been resolved.
```
From:	YW-Dev@<domain>.com
Sent:	Tuesday, April 13, 2021 9:54 PM
To:	YugabyteOperations
Subject:	Yugabyte Platform Alert - <[yw-dev][dev]>

Clock Skew Alert for <Universe_Name> is resolved.
```
When examining the application logs we do not see any indication of a clock skew or any hybrid time issues. 

Please reference ticket: [1045](https://yugabyte.zendesk.com/agent/tickets/1045)

**Comment by CharlotteRose** [comment]
UPDATE: 

There are now over 1700 email alerts in just a few minutes now in PRODUCTION after the 2.4.3 upgrade.
The user has "snoozed" alerts on the PROD universe for now.

Please reference ticket: [1045](https://yugabyte.zendesk.com/agent/tickets/1045)

**Comment by SergeyPotachev** [comment]
@CharlotteRose 

Health check doesn't check the actual clock skew, it only checks that NTP time synchronization is turned ON.
And the alert is generated according to metric values exposed by tserver/master processes.

For now we have two possible solutions for this -
1. In `reference.conf` we have a configuration parameter for this:
```
    # Value of maximum allowed clock skew before an alert is generated (in ms).
   yb.alert.max_clock_skew_ms = 500

```
We can increase it, but at first we need to understand what is the value of the metric (from the Prometheus metrics page).
But if this is a replicated installation (I think it is) then it will be dropped to the default value in case of each container restart.

2. We can disable only this alert type directly in DB (let me know if you need the SQL query) or we can set/increase the alert threshold directly in DB (this will survive in case of the containers restarts).

**Comment by streddy-yb** [comment]
hi @SergeyPotachev - Let us provide a config to disable clock skew alerts similar to backup alerts (screenshot below) in case user wants to turn off the alerts.

![image](https://user-images.githubusercontent.com/68868569/119279914-67e58f00-bbe3-11eb-9a91-c9a1f3dddd3f.png)

**Comment by SergeyPotachev** [comment]
@streddy-yb Yes, sure. Actually #8212 is planned to have an ability to turn on/off different alerts separately.
